plugins/Android/src/net/gree/unitywebview/CWebViewPlugin.java

```diff
@@ -132,6 +142,19 @@ public void call(final String method, final String message) {
     public CWebViewPlugin() {
     }
 
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String permissions[], int[] grantResults) {
+        switch (requestCode) {
+            case REQUEST_CODE:
+                if (grantResults[0] == PackageManager.PERMISSION_GRANTED) {
```

Comment:
It seems `grantResults` could be zero length array.
In our project we are receiving sometimes crashes related to this method.
![image](https://user-images.githubusercontent.com/70874831/149302418-6a7880a7-3412-4b61-bc72-17eb19db15f1.png)